pages/training.php

```diff
@@ -27,12 +27,15 @@ function training() {
 
 	// Forbidden seal trainings boost
 	if($player->forbidden_seal && $player->forbidden_seal['level'] >= 2) {
+        $long_boosts = System::$premium_benefits[$player->forbidden_seal['level']]['enhanced_long_training'];
+        $extended_boosts = System::$premium_benefits[$player->forbidden_seal['level']]['enhanced_extended_training'];
```

Comment:
With the previous suggestion, this becomes
```suggestion
        $long_boosts = $player->forbidden_seal->enhanced_long_training;
        $extended_boosts = $player->forbidden_seal->enhanced_extended_training;
```